### ISSUE TITLE
TR1X bug: First backdrop flicker during fade-in when starting up the game

### ISSUE THREAD

**Issue by aredfan** [LostArtefacts/TRX#1388]
**Dev snapshot:** 4.1.2-45-g5d706bd

______________________

When starting up the game - the first visible backdrop flickers during fade-in.  A sort-of black blink.

I think this regression was introduced today.

**Steps to reproduce:**

- Disable FMVs.
- Set FPS to 30 before starting up the game.  60 FPS is affected too but it's hard to tell.
- If eidos logo is enabled then that particular backdrop is affected.
- If eidos logo is disabled then the main menu backdrop is affected instead.

___________________

**Edit:** The issue happens when vsync is enabled.

**Comment by rr-** [comment]
@aredfan I can't quite see this, can you show some exaggerated video and post the config file? Thanks.

**Comment by aredfan** [comment]
@rr- Yes of course, here's my config files: [cfg.zip](https://github.com/user-attachments/files/15987352/cfg.zip).

Btw I did more investigating today and found that the blink is a lot more apparent when `Fade effects` is disabled.  Maybe the bug isn't related to fades.

My screen-recorder GeForce wasn't able to capture the issue so I recorded it on my phone instead, it's slow motion.  See the Eidos screen appear for a frame or two, before properly appearing.

The video is recorded with fades disabled, vsync on, and 30 FPS.

https://github.com/LostArtefacts/TR1X/assets/94776985/73ad9a1d-f94b-43c0-b219-c8f920422a0d

**Comment by aredfan** [comment]
I thought it might be a good idea to write a short summary of what we know so far.

- The first backdrop to appear is affected by this issue.  It can be the Eidos screen.  If the Eidos screen and FMVs are disabled then the main menu backdrop is affected instead.
 - The flickering is noticeable when vsync is on.  It's possible the flicker is present when vysnc is off but it's very hard to tell.
- When I disable fades, it makes the flicker easier to see.
- The flicker only happens in fullscreen and not windowed mode, it happens regardless of render mode.

**Comment by rr-** [comment]
@aredfan can you check if this build makes any difference?

https://nebula.wind.garden/serve/pub/TR1X-4.1.2-62-gc034908-dirty-Windows.zip

**Comment by aredfan** [comment]
@rr- I just tested the build and the flicker is still present.

**Comment by aredfan** [comment]
This issue was fixed at some point in develop even without the nvidia workaround mentioned in 635.
